tsconfig.json

```diff
@@ -24,7 +24,7 @@
     // "useDefineForClassFields": true,                  /* Emit ECMAScript-standard-compliant class fields. */
 
     /* Modules */
-    "module": "ES2020",                                /* Specify what module code is generated. */
+    "module": "CommonJS",                                /* Specify what module code is generated. */
```

Comment:
Curious: what prompted the change?

Reply:
I changed it to follow the setup typescript and express api article.  I don't have a preference on what we have our compile module.  With these changes everything should run fine. if we wanted to compile to ESM it would require more changes.